docs/evaluate/temporal-cloud/legacy-support.mdx

```diff
@@ -0,0 +1,160 @@
+Our team has extensive knowledge of Temporal, and a broad set of skills to help you succeed with any project.
+
+Temporal Cloud provides several levels of support, from assisting with break/fix scenarios to helping with onboarding, design/code reviews for your application, optimizations, and operational readiness.
+
+:::note
+
+The content of this page applies to Temporal Cloud customers only.
+
+:::
+
+## What are the services offered by Temporal Cloud? {#services}
+
+We offer four services to Temporal Cloud customers.
+For access to any of these services, please [create a support ticket](/cloud/support#support-ticket) or contact your dedicated account manager.
+
+- **Technical onboarding:** Temporal Cloud customers can request an onboarding session.
+  This session covers basic setup and Namespace planning and provides an overview of metrics and observability, an overview of development resources, and information you will need in preparation for your go-live.
+- **Design and code reviews:** Our team works with you to review your Workflow design and code to avoid potential issues and optimize your workload to run on Temporal Cloud.
+  These sessions also focus on the optimization of your Workflow Actions and reduce your costs; they also reveal likely trade offs.
+
+## What is the Temporal Cloud support guarantee? {#support}
+
+Temporal endeavors to ensure you are successful with Temporal Cloud.
+We offer explicit guarantees for support.
+Temporal Cloud customers get break/fix support with an agreed-upon set of SLAs for prioritized issues.
+We use a ticketing system for entering, tracking, and closing these issues.
+
+If an issue occurs, the team also provides support through a dedicated Slack channel, forums, and a knowledge base.
+We offer two levels of support defined by their availability and SLAs in the following table:
+
+|                          | **Basic**                                                                                      | **Enterprise**                                                                            |
+| ------------------------ | ---------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
+| **Response times**       | P0: 1 business hour<br />P1: 4 business hours<br />P2: 1 business day<br />P3: 2 business days | P0: 30 minutes<br />P1: 1 business hour<br />P2: 4 business hours<br />P3: 1 business day |
+| **Onboarding & Success** | Development, deployment, and optimization guidance                                             | Development, deployment, and optimization guidance                                        |
+| **Channels**             | Community Slack, Email, Forums, Documentation, Knowledge Base                                  | Private Slack Channel, Email, Forums, Documentation, Knowledge Base                       |
+
+:::info BUSINESS HOUR TIMEZONES
```

Comment:
Do we capitalize these? I don't think we do.

Reply:
This is automatically capped in rendering. Putting it in caps in the mdx reflects how it will be rendered.

Reply:
Lower casing these and pushing.